.github/workflows/tests.yml

```diff
@@ -117,6 +124,38 @@ jobs:
           nox -s a11y
         continue-on-error: true
 
+  coverage:
+    name: Collect Coverage
+    runs-on: ubuntu-latest
+    needs: run-pytest
+    # run both on previous step success and failure
+    if: "!cancelled()"
```

Comment:
I'm assuming we want coverage even if tests are failing, but not if workflow is cancelled.